apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.scss

```diff
@@ -0,0 +1,3 @@
+.error-box.select-error {
```

Comment:
Remove this as it will make red error border to hide as it is more specific or please test it for error scenario

Reply:
I discussed with Rohit about this error red color,  Rohit will work on in it